content/docs/Konveyor/Kai/RunAnalysis/proc_running-agent-analysis.md

```
@@ -0,0 +1,50 @@
+---
+title: "Generating code resolutions in the agent mode"
+date: 2025-11-30T14:58:52-06:00
+draft: false
+---
+In the agent mode, the Konveyor AI planning agent creates the context for an issue and picks a sub-agent that is most suited to resolve the issue. The sub-agent runs an automated scan to describe how the issue can be resolved and generates files with the updated resolutions in one stream.
+
+You can review the updated files and approve or reject the changes to the code. The agent runs another automated analysis to detect new issues in the code that may have occurred because of the accepted changes or diagnostic issues that your tool may generate following a previous analysis. If you allow the process to continue, Konveyor runs the stream again and generates a new file with the latest updates.
+
+When using the agent mode, you can reject the changes or discontinue the stream but you cannot edit the updated files during the stream.
+
+**Prerequisites**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)